schema.json

```diff
@@ -1735,7 +1763,7 @@
         "force_reload": true,
         "options": [
           {
-            "value": "1280x1280",
+            "value": "1920x1920",
```

Comment:
This isn't completely necessary to realize the value of responsive images, but I like having the zoom be a little zoomier :)